### DISCUSSION TITLE
Session.secure

### DISCUSSION THREAD

**Discussion by BenMilanzi98** [devchopra999/Whatsapp-bot#5] [General]
Hello, I have followed everything ith deploy guide. unfortunately am having the following problem
![Screenshot (118)](https://user-images.githubusercontent.com/36082995/167900707-2a0b8246-8112-4787-9cc8-056f1095dd07.png)
:

1. The session secure file is about 4.8mb and github does not support viewing raw files of that size.
what can be the solution for this? (each time i press on view raw, it downloads the file back to my pc)

**Comment by devchopra999** [comment]
Hello,
you just have to right-click on `view raw` and click on copy link address
this is your session_url that you must paste in heroku

[Telegram Support Channel](https://t.me/SciBot_Whatsapp)

**Reply by BenMilanzi98** [reply]
Great. let me try that
